windows/vtpm_cold_add_remove/vtpm_add_remove_check.yml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+- name: "Get keyId info in VM config"
+  include_tasks: ../../common/vm_get_cryptokey_id.yml
+
+# Check VM is encrypted after adding vTPM device
+- name: "Make sure keyId is set in VM config"
+  ansible.builtin.assert:
+    that:
+      - vm_cryptokey_id_info
+      - vm_cryptokey_id_info.keyId is defined
+      - vm_cryptokey_id_info.keyId
+      - vm_cryptokey_id_info.providerId is defined
+      - vm_cryptokey_id_info.providerId.id is defined
+      - vm_cryptokey_id_info.providerId.id
+    fail_msg: "'keyId' is not set in VM config: {{ vm_cryptokey_id_info }}, which should be set after adding vTPM device."
+  when: vtpm_test_operation == 'add'
+
+# Check VM is not encrypted after removing vTPM device
+- name: "Check keyId is unset in VM config"
+  ansible.builtin.assert:
+    that:
+      - not vm_cryptokey_id_info
+    fail_msg: "'keyId' is set in VM config: {{ vm_cryptokey_id_info }}, which should be unset after removing vTPM device."
+  when:
+    - vtpm_test_operation == 'remove'
+    - vm_encrypt_state_before != "encrypt"
```

Comment:
The encrypted VM before adding vTPM cannot be decrypted even after removing vTPM?  It means the keyId cannot be changed when adding vTPM to it, right?

Reply:
Both the encrypted VM and the VM with vTPM device have the 'KeyId' info in VM config.